app.js

```diff
@@ -42,9 +47,41 @@ app.on("activate", function() {
     }
 });
 
+// Register updater listeners.
+autoUpdater.on("checking-for-update", () => {
+    logger.info("Checking for update...");
+});
+
+autoUpdater.on("update-available", () => {
+    logger.info("Update available.");
+});
+
+autoUpdater.on("update-not-available", () => {
+    logger.info("Update not available.");
+});
+
+autoUpdater.on("error", (err) => {
+    logger.info("Error in auto-updater. " + err);
+});
+
+autoUpdater.on("download-progress", (progressObj) => {
+    let msg = "Download speed: " + progressObj.bytesPerSecond;
+    msg += " - Downloaded " + progressObj.percent + "%";
```

Comment:
I *think* you can use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for string interpolation here and elsewhere:

```
` - Downloaded ${progressObj.percent}%`
```

Reply:
Indeed! Changed it.